utils/common/src/main/kotlin/MapUtils.kt

```diff
@@ -21,6 +21,14 @@ package org.ossreviewtoolkit.utils.common
 
 import java.util.SortedMap
 
+/**
+ * Return the set of keys that are present in both maps if maps are not equal, otherwise return the empty set.
+ */
+fun <K, V> Map<K, V>.getConflictingKeys(other: Map<K, V>): Set<K> {
+    if (this == other) return emptySet()
+    return keys.intersect(other.keys)
```

Comment:
Shouldn't this return only keys where the value is not equal?

Reply:
So basically `getConflictingKeys` would of `{"a": "a", "b": "b"}` and `{"a": "a", "b": "other"}` would return only `b`.

Reply:
Otherwise your `--project-scanners FossId --package-scanners ScanCode` will certainly also be problematic.